### ISSUE TITLE
 ERROR Error: ASSERTION ERROR: Reached the max number of directives  Angular v9.05

### ISSUE THREAD

**Issue by CerealKiller97** [davidwalschots/angular-reactive-validation#3]
Hi,

I am getting this error. This happens when I put arv-messages component to diplay errors. Can you upgrade this lib for Angular 9 support?

Thanks in advance.
```
 ERROR Error: ASSERTION ERROR: Reached the max number of directives [Expected=> 4 != 4 <=Actual]
    at throwError (core.js:1309)
    at assertNotEqual (core.js:1265)
    at initTNodeFlags (core.js:12915)
    at resolveDirectives (core.js:12598)
    at elementStartFirstCreatePass (core.js:20979)
    at Module.ɵɵelementStart (core.js:21020)
    at LoginComponent_Template (login.component.html:11)
    at executeTemplate (core.js:11926)
    at renderView (core.js:11712)
    at renderComponent (core.js:13228)
    at renderChildComponents (core.js:11515)
    at renderView (core.js:11738)
    at ComponentFactory$1.create (core.js:33909)
    at ViewContainerRef.createComponent (core.js:15376)
    at RouterOutlet.activateWith (router.js:8939)
    at RouterOutlet.ngOnInit (router.js:8846)
    at callHook (core.js:3935)
    at callHooks (core.js:3899)
    at executeInitAndCheckHooks (core.js:3840)
    at refreshView (core.js:11791)
    at refreshComponent (core.js:13213)
    at refreshChildComponents (core.js:11504)
    at refreshView (core.js:11825)
    at refreshDynamicEmbeddedViews (core.js:13138)
    at refreshView (core.js:11796)
    at refreshComponent (core.js:13213)
    at refreshChildComponents (core.js:11504)
    at refreshView (core.js:11825)
    at renderComponentOrTemplate (core.js:11899)
    at tickRootContext (core.js:13375)
    at detectChangesInRootView (core.js:13409)
    at RootViewRef.detectChanges (core.js:15089)
    at ApplicationRef.tick (core.js:42697)
    at core.js:42547
    at ZoneDelegate.invoke (zone-evergreen.js:365)
    at Object.onInvoke (core.js:41271)
    at ZoneDelegate.invoke (zone-evergreen.js:364)
    at Zone.run (zone-evergreen.js:124)
    at NgZone.run (core.js:41046)
    at Object.next (core.js:42544)
    at SafeSubscriber.schedulerFn [as _next] (core.js:36874)
    at SafeSubscriber.__tryOrUnsub (Subscriber.js:183)
    at SafeSubscriber.next (Subscriber.js:122)
    at Subscriber._next (Subscriber.js:72)
    at Subscriber.next (Subscriber.js:49)
    at EventEmitter.next (Subject.js:39)
    at EventEmitter.emit (core.js:36836)
    at checkStable (core.js:41185)
    at Object.onHasTask (core.js:41292)
    at ZoneDelegate.hasTask (zone-evergreen.js:420)
```

**Comment by davidwalschots** [comment]
@CerealKiller97 I've released version 4.0.1 of the package today. It should be compatible with Angular 9. While working on Angular 9 compatibility I didn't come across the issue you mention here. It could be it is unrelated to the package.

If you have any more problems, please let me know and add a code sample that can reproduce the problem.

**Comment by CerealKiller97** [comment]
@davidwalschots thank you, a lot it works perfectly, you made my day! Dank je wel!
